.github/workflows/callable-e2e-tests.yml

```diff
@@ -0,0 +1,111 @@
+name: 'E2E Tests'
```

Comment:
This file looks like replaces `callable-e2e-example.yml`. You can probably delete `callable-e2e-example.yml` as dead code, right?

Reply:
makes sense, updated